src/components/CustomLoader.tsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import ContentLoader from "react-content-loader"
 import { Card } from 'reactstrap';
 
-const CustomLoader = props => (
+const CustomLoader = (props: any) => (
```

Comment:
I will create a separate issue for types. No worries

Reply:
Cool!